pyghee/lib.py

```diff
@@ -42,7 +41,7 @@ def get_event_info(request):
         'raw_request_headers': dict(request.headers),
     })
 
-    timestamp = datetime.datetime.utcfromtimestamp(int(event_info['timestamp_raw'])/1000.)
+    timestamp = datetime.datetime.fromtimestamp(int(event_info['timestamp_raw'])/1000., datetime.UTC)
```

Comment:
The `datetime` documentation recommends to use `timezone.utc` here, so let's do so?

```suggestion
    timestamp = datetime.datetime.fromtimestamp(int(event_info['timestamp_raw'])/1000., tz=datetime.UTC)
```

Reply:
`datetime.UTC` is an alias for `datetime.timezone.utc`
https://docs.python.org/3/library/datetime.html#datetime.UTC

Reply:
OK, makes sense, but I'm still in favor of adding the `tz=` to make it explicit we're setting an optional value

Reply:
Updated 👍